cranelift/codegen/src/isa/aarch64/inst/regs.rs

```diff
@@ -176,8 +176,7 @@ fn show_reg(reg: Reg) -> String {
     }
 }
 
-pub fn pretty_print_reg(reg: Reg, allocs: &mut AllocationConsumer) -> String {
-    let reg = allocs.next(reg);
+pub fn pretty_print_reg(reg: Reg, _allocs: &mut AllocationConsumer) -> String {
```

Comment:
Is any of these functions that take an `AllocationConsumer` to pretty print actually using that parameter, or just some of them?

Reply:
Right, none of the `AllocationConsumer` arguments are actually necessary, after #8524 landed. I'm deleting all the newly dead code in several steps to make review easier, so this PR deletes all remaining calls to `AllocationConsumer` methods.

There are still some functions where the Rust compiler thinks the `AllocationConsumer` argument is used, but that's because it's passed to other functions that don't actually need it either. I intend to delete all those arguments, and all mentions of the `AllocationConsumer` type, in another PR after this one.